src/CronTask.php

```diff
@@ -2004,4 +2004,46 @@ public static function getIcon()
     {
         return "ti ti-settings-automation";
     }
+
+    /**
+     * @return void
+     * @used-by templates/components/search/controls.html.twig
+     */
+    public static function showSearchStatusArea()
+    {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
+
+        $crontask = new self();
+        $warnings = [];
+        if ($crontask->getNeedToRun(self::MODE_INTERNAL)) {
+            $warnings[] = __("You have at least one automatic action configured in GLPI mode, we advise you to switch to CLI mode.");
+        }
+
+        if (
+            $CFG_GLPI['cron_limit'] < countElementsInTable(
+                'glpi_crontasks',
+                ['frequency' => MINUTE_TIMESTAMP]
+            )
+        ) {
+            $warnings[] = __("You have more actions scheduled to run every minute than the number allowed to run at once. Increase this config.");
+        }
+
+        if (count($warnings) > 0) {
+            $msg = __s('Automatic actions may not be running as expected');
```

Comment:
Will be escaped by Twig engine.
```suggestion
            $msg = __('Automatic actions may not be running as expected');
```